src/rp2040/boot_stage2/CMakeLists.txt

```diff
@@ -33,7 +33,9 @@ set(PICO_BOOT_STAGE2_DIR "${CMAKE_CURRENT_LIST_DIR}" CACHE INTERNAL "")
 pico_add_library(boot_stage2_headers)
 target_include_directories(boot_stage2_headers SYSTEM INTERFACE ${CMAKE_CURRENT_LIST_DIR}/include)
 
-# by convention the first source file name without extension is used for the binary info name
+# pico_define_boot_stage2(TARGET NAME SOURCES)
```

Comment:
Possibly a silly question, but why does this line say `pico_define_boot_stage2(TARGET NAME SOURCES)` when a few lines below it says `function(pico_define_boot_stage2 NAME SOURCES)`? Note that `TARGET` is listed in one but not the other.
(same thing is true for some other new descriptions in this PR too)

Reply:
That's a mistake - will fix it

Reply:
I think this is just a typo for the boot_stage2 descriptions (rp2040 & rp2350) - let me know if you spot any others

Reply:
Ah - if i's just a typo, perhaps i's worth expanding your Python regexes so that you can check the description-signature matches the actual cmake-function-signature? :slightly_smiling_face:

Reply:
The issue it that it doesn't always match when doing argument parsing - it's probably possible to check the first argument or something, but I think I'll leave that for a separate PR